### ISSUE TITLE
Unable to join or watch any games except bot games since update

### ISSUE THREAD

**Issue by kaffle** [ValveSoftware/Dota2-Gameplay#12237]
### Description

"Unable to Ping any Region" error persists when attempting matchmaking. When selecting "Refresh Pings", client attempts indefinitely to calculate pings. 

When attempting to watch live games, game status says "Connecting" and attempts to join before being booted back to the same menu. 

No changes to network or firewall were made prior to update. No VPN in use. 

### Screenshots

![error](https://github.com/ValveSoftware/Dota2-Gameplay/assets/146298260/84d63d2d-78d0-4d1e-9a45-048186c8c97d)

**Comment by Emily358** [comment]
same issue here, no firewall changes or vpn. my friend is connecting on the same wifi as me just fine so its not an internet issue. got timeouts for trying to join a game because it just kept booting me.

**Comment by patrick-andrew-jain-taylor** [comment]
Same thing happening for me on both GeForce Now and Boosteroid.

**Comment by kaffle** [comment]
> Same thing happening for me on both GeForce Now and Boosteroid.

I was able to fix it by restarting my PC which seems very obvious. I'm curious if others who are having the same issue have tried this. Although I doubt streaming PCs such as on GeForce Now and Boosteroid can be fixed until their host PCs are reset

**Comment by AnatoliyLitinskiy** [comment]
same issue. Ping never ends calculating. Unable to connect.

**Comment by Craftiii4** [comment]
Same here, tried restarting computer and router, tried on mobile hot-spot, tried with a VPN, but nothing worked.

**Comment by zakhad** [comment]
Yeah, same for me i haven't been able to play, have restarted router, restarted pc etc
